CONTRIBUTING_CODE.md

```diff
@@ -266,21 +270,35 @@ Use the following information to use Spark for code testing.
 
 - Java. See [Java downloads](https://www.java.com/en/download/).
 
-- The PATH or JAVA_HOME environment variables set correctly. See [Setting Java variables in Windows](https://www.ibm.com/docs/en/b2b-integrator/5.2?topic=installation-setting-java-variables-in-windows) or [Setting Java variables in Linux](https://www.ibm.com/docs/en/b2b-integrator/5.2?topic=installation-setting-java-variables-in-linux).
+- The PATH or JAVA_HOME environment variables set correctly. See [Setting Java variables in Windows](https://www.ibm.com/docs/en/b2b-integrator/5.2?topic=installation-setting-java-variables-in-windows) or [Setting Java variables in Linux](https://www.ibm.com/docs/en/b2b-integrator/5.2?topic=installation-setting-java-variables-in-linux). 
+
+On Mac, if you installed Java via the download above, you can set PATH and JAVA_HOME using:
+```
+export JAVA_HOME=`/usr/libexec/java_home`
+export PATH=$PATH:$JAVA_HOME/bin 
+```
 
-- Apache Spark installed and configured correctly. See [Spark Overview](https://spark.apache.org/docs/latest/index.html#downloading).
 
 #### Install PySpark
 
-Run the following command to install the PySpark version that matches your version of Apache Spark:
+- Installing PySpark will now install Spark for you. See [Spark Overview](https://spark.apache.org/docs/latest/index.html#downloading).
 
-```console
-spark-submit --version 2>&1 | grep -o "version [0-9][^ ]*$"
-version 3.2.0
+Run the following command to install the PySpark & Apache Spark:
 
-pip install "pyspark==3.2.0"
+```console
+pip install pyspark
 ```
 
+## If you want to build documentation locally:
+
+`pip install -r docs/requirements.txt`
```

Comment:
I think these instructions are for the legacy docs (in `docs_rtd`). We should probably replace these instructions with instructions on how to build our current docs. I'm hoping to find a bit of time today to simplify that process using an invoke command - let's coordinate.

Reply:
Ah sounds good. Looks like the docs here already link to the README within docs folder. I will remove this section that is no longer required.

Reply:
Awesome! For reference, here is the open PR with a new method to build docs locally and updated instructions: https://github.com/great-expectations/great_expectations/pull/7892